db/migrate/20240420104318_add_title_desc_user_id_to_api_keys.rb

```diff
@@ -0,0 +1,8 @@
+class AddTitleDescUserIdToApiKeys < ActiveRecord::Migration[6.1]
+  def change
+    add_column :api_keys, :title, :string
+    add_column :api_keys, :description, :string
+    add_column :api_keys, :user_id, :integer
+    add_column :api_keys, :team_id, :integer
```

Comment:
I think it's better to add the new columns as `reference` other than `integer`... any reason for not doing so?